exporter/elasticsearchexporter/exporter.go

```diff
@@ -130,6 +130,10 @@ func (e *elasticsearchExporter) pushLogsData(ctx context.Context, ld plog.Logs)
 						return cerr
 					}
 
+					if errors.Is(err, ErrInvalidTypeForBodyMapMode) {
+						errs = append(errs, fmt.Errorf("dropping log record: %w", err))
+					}
```

Comment:
sorry, my PR just merged, it is this one: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35740
Instead of appending the err, just log and drop it.